### PR TITLE
Test/filesystem local test

### PR DESCRIPTION
**Pull request by kasvith**
Pull Request for Issue #189  .

### Summary of Changes
- Added PHPUnit test for FileSystemLocal plugin
- Removed old LocalAdapterTest from tests

### Testing Instructions
- run `libraries\vendor\phpunit --testsuit plugins`

### Expected result
- Pass all test cases

### Actual result
- Pass all test cases

### CODE REVIEW

**Comment by laoneo** [comment]
You should move the existing file to the new location and not copy it.

**Comment by kasvith** [comment]
Used new plugin instead in new unit test

File: tests/unit/suites/plugins/filesystem/local/PlgFileSystemLocalTest.php [hunk 1]
```diff
@@ -7,14 +7,16 @@
  * @license     GNU General Public License version 2 or later; see LICENSE
  */
 
+require JPATH_BASE . '/plugins/filesystem/local/local.php';
```
Comment (laoneo) [inline tests/unit/suites/plugins/filesystem/local/PlgFileSystemLocalTest.php:10]: use JLoader

File: tests/unit/suites/plugins/filesystem/local/PlgFileSystemLocalTest.php [hunk 2]
```diff
@@ -7,14 +7,16 @@
...
+
 /**
  * Test class for local file adapter.
  *
  * @package     Joomla.UnitTest
  * @subpackage  com_media
  * @since       __DEPLOY_VERSION__
  */
-class LocalAdapterTest extends TestCaseDatabase
+class PlgFileSystemLocalTest extends TestCaseDatabase
```
Comment (laoneo) [inline tests/unit/suites/plugins/filesystem/local/PlgFileSystemLocalTest.php:19]: LocalAdapterTest is fine as name

File: tests/unit/suites/plugins/filesystem/local/PlgFileSystemLocalTest.php [hunk 3]
```diff
@@ -70,7 +88,10 @@ public function testGetFile()
 		JFile::write($this->root . 'test.txt', 'test');
 
 		// Create the adapter
-		$adapter = new MediaFileAdapterLocal($this->root);
+		$adapter = $this->class->onFileSystemGetAdapters($this->root);
```
Comment (laoneo) [inline tests/unit/suites/plugins/filesystem/local/PlgFileSystemLocalTest.php:91]: Plugin onFileSystemGetAdapters tests should be in it's own test class.

**Review by laoneo — CHANGES_REQUESTED** [review]

File: tests/unit/suites/plugins/filesystem/local/PlgFileSystemLocalTest.php (hunk 3 above)
Comment (kasvith) [reply tests/unit/suites/plugins/filesystem/local/PlgFileSystemLocalTest.php:91]: yes working on it

**Pull request closed without merging**